cmd/nerdctl/stats.go

```diff
@@ -313,9 +313,6 @@ func statsAction(cmd *cobra.Command, args []string) error {
 			return errors.New("unsupported format: \"raw\"")
 		default:
 			tmpl, err = parseTemplate(format)
-			if err != nil {
-				break
-			}
```

Comment:
What should we do with this err?
Cc @fahedouch

Reply:
what is the problem with this err @manugupt1 ?

Reply:
It does not break out the ticker loop and therefore, it is ineffective. Should I modify the code to stop the ticker?
Here is an example: https://go.dev/play/p/vpSz8gWxyQr

Reply:
We should parse the template and check the err before starting the ticker, but it can be another PR.